### ISSUE TITLE
Fonts in the dictionary do not display correctly

### ISSUE THREAD

**Issue by jiahe224** [goldendict/goldendict#946]
Fonts in the dictionary do not display correctly, Dictionary format is mdx and mdd，but Mdict can be displayed correctly，windows 10.0.15063 64bit，GoldenDict 1.5.0-RC2-209-gfe9312e

**Comment by jiahe224** [comment]
anyone？

**Comment by Abs62** [comment]
How about the problem dictionary example?

**Comment by jiahe224** [comment]
@Abs62 thank you reply。
please open the link：https://pan.baidu.com/s/1d1RRaA   keyword：wpu4

**Comment by Abs62** [comment]
Can't download. Can you upload sample to some other file sharing service?

**Comment by jiahe224** [comment]
@Abs62 https://drive.google.com/open?id=1FPDHQ0bxqOIoRrIn1qtBd6e-MVfXsDK-, Mdict:http://www.mdict.cn/download/beta/MDictPC.7z

**Comment by Abs62** [comment]
The resource file name in you sample don't match the dictionary file name (".1" in it is extraneous). And I don't see any font files in the resource file. Articles use fonts described in oalecd8e.css/O8C.css file. Check computed style for elements via GD inspector.

**Comment by jiahe224** [comment]
@Abs62 Yes, mdx dictionary font style determined by the css file, the computer has installed the relevant fonts, but  font style can not be properly displayed in the GoldenDict, Mdict can be displayed correctly

**Comment by Abs62** [comment]
Check "font-family" parameter in resulted style for elements with wrong font (via GD inspector). Try to save (complete) article and view it in browser.
